skype/skype-ps/skype/New-CsHuntGroup.md

```diff
@@ -13,14 +13,11 @@ Creates new Call Queue in your Skype for Business Online organization. Call Queu
 ## SYNTAX
 
 ```
-New-CsHuntGroup [-Domain <Object>] [-Name <Object>] [-AgentAlertTime <Object>]
- [-BypassDualWrite <Object>] [-ClientAudience <Object>] [-DistributionLists <Object>]
- [-DomainController <Object>] [-Force] [-HuntGroupServiceCallbackUri <Object>] [-LineUri <Object>]
- [-MusicOnHoldFileContent <Object>] [-MusicOnHoldFileName <Object>] [-OverflowAction <Object>]
- [-OverflowActionTarget <Object>] [-OverflowThreshold <Object>] [-Ring <Object>]
- [-Tenant <Object>] [-TimeoutAction <Object>] [-TimeoutActionTarget <Object>] [-TimeoutThreshold <Object>]
- [-UseDefaultMusicOnHold <Object>] [-WelcomeMusicFileContent <Object>] [-WelcomeMusicFileName <Object>]
- [-AsJob] [<CommonParameters>]
+New-CsHuntGroup -Domain <Object> -Name <Object> [-AgentAlertTime <Object>] [-DistributionLists <Object>]
+ [-LineUri <Object>] [-MusicOnHoldFileContent <Object>] [-MusicOnHoldFileName <Object>] [-OverflowAction <Object>]
+ [-OverflowActionTarget <Object>] [-OverflowThreshold <Object>]  [-TimeoutAction <Object>] 
```

Comment:
Extra space before TimeOutAction, TimeOutActionTarget and WelcomeMusicFileContent, which is visible in the preview.